Content.Shared/Containers/ItemSlot/ItemSlotsSystem.cs

```diff
@@ -125,6 +125,16 @@ public void RemoveItemSlot(EntityUid uid, ItemSlot slot, ItemSlotsComponent? ite
             else
                 Dirty(itemSlots);
         }
+
+        public bool TryGetSlotById(EntityUid uid, string slotId, [NotNullWhen(true)] out ItemSlot? itemSlot, ItemSlotsComponent? component = null)
```

Comment:
```suggestion
        public bool TryGetSlot(EntityUid uid, string slotId, [NotNullWhen(true)] out ItemSlot? itemSlot, ItemSlotsComponent? component = null)
```
None of the other slot methods here have the "ById" suffix. This one shouldn't either since there's nothing else you could possibly use to get a slot.

Reply:
this was added in another PR and i assume duplicated here and i missed it, but i do agree with this review that it should be renamed